cms/djangoapps/azure_video_pipeline/utils.py

```
@@ -0,0 +1,93 @@
+from django.conf import settings
+
+from .media_service import LocatorTypes, MediaServiceClient
+from .models import AzureOrgProfile
+
+
+def get_azure_config(organization):
+    azure_config = {}
```

You might want to raise an exception in this function if the settings values aren't set.
Also, this should probably be configurable on a per-site basis since we are using the django sites framework. Anytime you check settings.FEATURES you should start by checking site_configurations

(for example)
enable_mktg_site = configuration_helpers.get_value(
        'ENABLE_MKTG_SITE',
        settings.FEATURES.get('ENABLE_MKTG_SITE', False)
    )

---

We planed to use configuration_helpers, but didn't implement it because all sites have the same CMS with the same URL.
Does it make sense to modify?

---

You're totally, disregard

---

*right